examples/dihedral/dihedral.yml

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env yason.py
+#
+# Combined MC and MD moves on a four-bead molecule
+# with 3 harmonic bonds, 2 harmonic bond angle, and 1 dihedral angle.
+#
+# You may disable either MD or MC to verify that they
+# individually give the same bead<->bead distribution
+#
+energy:
+    - bonded: {}
+
+atomlist:
+    - a: {mw: 1, dp: 5}
+    - b: {mw: 1, dp: 5}
+    - c: {mw: 1, dp: 5}
+    - d: {mw: 1, dp: 5}
+
+moleculelist:
+    - fourbody:
+        atomic: false
+        atoms: [a, b, c, d]
+        structure:
+          - a: [-9.80, -3.11, 7.57]
+          - b: [-9.56, -3.85, 8.26]
+          - c: [-8.10, -4.43, 7.78]
+          - d: [-8.38, -3.05, 7.16]
+        bondlist:
+          - harmonic: {k: 10, req: 1, index: [0,1]}
+          - harmonic: {k: 10, req: 1, index: [1,2]}
+          - harmonic: {k: 10, req: 1, index: [2,3]}
+          - harmonic_torsion: {k: 10, aeq: 90, index: [0,1,2]}
+          - harmonic_torsion: {k: 10, aeq: 45, index: [1,2,3]}
+          - periodic_dihedral: {k: 5, n: 2, phi: 180, index: [0,1,2,3]}
+
+insertmolecules:
+    - fourbody: {N: 1}
+
+moves:
+    - langevin_dynamics: # Langevin dynamics
```

Comment:
comment is superfluous due to excellent naming of keyword 👍🏻

Reply:
Fixed with https://github.com/mlund/faunus/pull/365/commits/8ff5d87f906a98ff380fb6b34381fbe46ddf24e3.